presto-main/src/test/java/com/facebook/presto/type/TestArrayOperators.java

```diff
@@ -160,6 +163,32 @@ public void testArrayElements()
         assertInvalidFunction("CAST(ARRAY ['puppies', 'kittens'] AS ARRAY<BIGINT>)", INVALID_CAST_ARGUMENT);
     }
 
+    private static String generateLongSring(int size, char c)
+    {
+        char[] cs = new char[size];
+        Arrays.fill(cs, c);
+
+        return new String(cs);
+    }
+
+    private static String generateThreeElementArrayString(int size)
+    {
+        String arrayDistinctSqlString = "array_distinct(ARRAY[" +
+                "'" + generateLongSring(size, 'x') + "', '" +
+                generateLongSring(size, 'y') + "', '" + generateLongSring(size, 'z') + "'])";
+
+        return arrayDistinctSqlString;
```

Comment:
I prefer to have return variables on their own line so i can set a breakpoint and see what the value is.

does this go against style guidelines?
(will update, but I'd like to have that discussion--often a large call stack unwinds and it's gnarly to see what a return value was)